lib/monkey_patches.rb

```diff
@@ -38,7 +38,20 @@ def process_full_diff
     end
   end
 
+  module LibMonkeyPatch
+    # Monkey patch set_custom_git_env_variables due to our ::Git::GitExecuteError handling.
+    #
+    # We rescue on the GitExecuteError and proceed differently based on the output of git.
+    # This way makes code language-dependent, so here we ensure that Git gem throw git commands with the "C" language
+    def set_custom_git_env_variables
+      super
+      ENV['LANG'] = 'C'
```

Comment:
I'm worried that this will break UTF-8 encoded values. Some distros build `C.UTF-8` but others don't. Any thoughts about this?

Reply:
Git uses `gettext` to translate its messages.
So you could set `LANG=C.UTF-8` to be sure to preserve UTF-8 encoded value, and on distros were `C.UTF-8` is not available it will fallback on untranslated message which will work in our case.

Are you OKay with this assertion?